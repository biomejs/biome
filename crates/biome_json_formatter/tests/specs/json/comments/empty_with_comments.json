{
    "object-block": { /* here's a block comment */},
    "object-line": { // here's a line comment
    },
    "object-ownline": {
        /* here's a block comment */
        // and a line comment
    },
    "array-block": [ /* here's a block comment */],
    "array-line": [ // here's a line comment
    ],
    "array-ownline": [
        /* here's a block comment */
        // and a line comment
    ]
}