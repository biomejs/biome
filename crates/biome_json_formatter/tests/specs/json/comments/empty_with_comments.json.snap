---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: json/comments/empty_with_comments.json
---

# Input

```json
{
    "object-block": { /* here's a block comment */},
    "object-line": { // here's a line comment
    },
    "object-ownline": {
        /* here's a block comment */
        // and a line comment
    },
    "array-block": [ /* here's a block comment */],
    "array-line": [ // here's a line comment
    ],
    "array-ownline": [
        /* here's a block comment */
        // and a line comment
    ]
}
```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Trailing commas: None
-----

```json
{
	"object-block": {
		/* here's a block comment */
	},
	"object-line": {
		// here's a line comment
	},
	"object-ownline": {
		/* here's a block comment */
		// and a line comment
	},
	"array-block": [
		/* here's a block comment */
	],
	"array-line": [
		// here's a line comment
	],
	"array-ownline": [
		/* here's a block comment */
		// and a line comment
	]
}
```

